docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -354,26 +358,26 @@ public class OidcDiscoveryRequestCustomizer implements OidcRequestFilter {
     }
 }
 ----
-<1> Restrict this filter to requests targeting the OIDC discovery endpoint only.
```

Comment:
IMHO it does not harm to clarify where exactly this enum is used

Reply:
Hi @sberyozkin, in your previous review of this topic before PTO, you proposed to remove this line 357 and I removed it and the corresponding callout number  "<1>". I'm not sure what to update there. Would you like me to re-insert this line or a modified version of it perhaps?

Reply:
@sheilamjones Oh, looks like I keep changing my mind :-), sorry. Yeah, it looks like it can be a useful piece of info, please put it back

Reply:
No worries. Thanks @sberyozkin. Re-inserted this line.